flutter/lib/src/event_processor/native_app_start_event_processor.dart

```diff
@@ -10,36 +12,24 @@ class NativeAppStartEventProcessor implements EventProcessor {
   /// We filter out App starts more than 60s
   static const _maxAppStartMillis = 60000;
 
-  NativeAppStartEventProcessor(
-    this._native,
-  );
+  final AppStartTracker? _appStartTracker;
 
-  final SentryNative _native;
+  NativeAppStartEventProcessor({
+    AppStartTracker? appStartTracker,
+  }) : _appStartTracker = appStartTracker ?? AppStartTracker();
+
+  bool didAddAppStartMeasurement = false;
```

Comment:
That's because `SentryNative.didFetchAppStart` doesn't tell us 'we've already added this measurement to the transaction`

since the event processor is always run for every event checking `if (didFetchAppStart == true)` and then adding the measurement would mean we are always adding measurement because after the app start it's always true.

we could also modify `sentry_native` so we can mutate this flag after we processed the first app start event so this doesn't happen